web/src/components/Core/Page.js

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react'
+
+import CircularProgress from '@material-ui/core/CircularProgress'
+import Fade from '@material-ui/core/Fade'
+import { makeStyles } from '@material-ui/core/styles'
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
are we making a pact to do no separate .scss files this time around?

Reply:
yup